test/pdfboxing/info_test.clj

```diff
@@ -6,15 +6,35 @@
   (testing "page count"
     (is (= 1 (page-number "test/pdfs/interactiveform.pdf"))))
   (testing "field count"
-    (is (= 6 (count (about-doc "test/pdfs/interactiveform.pdf")))))
+    (is (= 10 (count (about-doc "test/pdfs/interactiveform.pdf")))))
   (testing "for the key names"
-    (is (= ["title" "author" "subject" "keywords" "creator" "producer"]
-           (keys (about-doc "test/pdfs/interactiveform.pdf")))))
+    (is (= #{"title" "author" "subject" "keywords" "creator" "producer" "trapped" "metadata-keys" "creation-date" "modification-date"}
+           (set (keys (about-doc "test/pdfs/interactiveform.pdf"))))))
   (testing "if the full document info matches"
     (is (= {"title" "Example of an Interactive PDF Form",
             "author" nil,
             "subject" "PDF forms",
             "keywords" "forms, flat, fillable",
+            "trapped" nil,
+            "metadata-keys" #{"AAPL:Keywords" "CreationDate" "Creator" "Keywords" "ModDate" "Producer" "Subject" "Title"}
             "creator" "Adobe InDesign CS3 (5.0.4)",
             "producer" "Mac OS X 10.8.5 Quartz PDFContext"}
-           (about-doc "test/pdfs/interactiveform.pdf")))))
+           (dissoc (about-doc "test/pdfs/interactiveform.pdf")
+                   "creation-date" ;#inst "2013-12-14T13:44:39.000+00:00"
+                   "modification-date" ;#inst "2013-12-14T13:44:39.000+00:00"
+                   ))))
+  (testing "for the metadata value"
+    (is (= "Example of an Interactive PDF Form"
+           (metadata-value "test/pdfs/interactiveform.pdf" "Title")))
+    (is (= "D:20131214134439Z00'00'"
+           (metadata-value "test/pdfs/interactiveform.pdf" "CreationDate")))
+    (is (= {"AAPL:Keywords" nil,
+            "CreationDate" "D:20131214134439Z00'00'",
+            "Creator" "Adobe InDesign CS3 (5.0.4)",
+            "Keywords" "forms, flat, fillable",
+            "ModDate" "D:20131214134439Z00'00'",
+            "Producer" "Mac OS X 10.8.5 Quartz PDFContext",
+            "Subject" "PDF forms",
+            "Title" "Example of an Interactive PDF Form"}
+           (metadata-values "test/pdfs/interactiveform.pdf")))
+    ))
```

Comment:
Can the trailing parentheses be moved up on the previous line as per 'The Clojure Style Guide':

https://github.com/bbatsov/clojure-style-guide#gather-trailing-parens